### PR TITLE
WIP Please review Sridhar dublin branch sridhar.md is added

### PR DESCRIPTION
**Pull request by sswaminathanart**
WIP Please review Sridhar dublin branch sridhar.md is added

Hi please review @senthil33; @santhimuthusamy

### CODE REVIEW

File: sridhar.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Sridhar
```
Comment (senthil33) [inline sridhar.md:1]: What town you are writing review for?

**Review by senthil33 — CHANGES_REQUESTED** [review]
Good. Need few changes

File: sridhar.md (hunk 1 above)
Comment (sswaminathanart) [reply sridhar.md:1]: i will do

**Review by sswaminathanart — COMMENTED** [review]
done

**Review by senthil33 — APPROVED** [review]

**Comment by govind999** [comment]
Everything looks good.

**Pull request merged**